enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/consensus/roles/Leader.java

```diff
@@ -307,9 +307,9 @@ public Outcome handle( LogCompactionInfo logCompactionInfo )
             return outcome;
         }
 
-        private void stepDownToFollower( Outcome outcome )
+        private void stepDownToFollower( Outcome outcome, long inTerm )
```

Comment:
Might be more readable (and safer) to take a `ReadableRaftState` rather than a `long`.